gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -130,6 +130,22 @@ internal open class AnvilPlugin : KotlinCompilerPluginSupportPlugin {
       "anvil${File.separator}src-gen-${variant.name}"
     )
 
+    if (variant.variantFilter.syncGeneratedSources) {
+      val isIdeSyncProvider = project.providers
+        .systemProperty("idea.sync.active")
+        .forUseAtConfigurationTime()
+
+      if (isIdeSyncProvider.getOrElse("false").toBoolean()) {
```

Comment:
From a CC perspective, this all looks fine.

Reply:
Thanks!